tests/core/test_config.py

```diff
@@ -385,6 +385,39 @@ def test_nested_group_value_badreg(config):
     assert "bar" in bar_list
 
 
+@pytest.mark.asyncio
+async def test_toggle(config):
+    config.register_global(foo=True)
+    value = await config.foo.toggle()
+    assert value is False
+
+
+@pytest.mark.asyncio
+async def test_inc_values(config):
+    defaults = {"bar": 0, "baz": {"subgroup": 10}}
+    config.register_global(foo=defaults)
+    negative = await config.foo.bar.inc(-5)
+    nested = await config.foo.baz.subgroup.inc(10)
+    assert negative == -5
+    assert nested == 20
+
+
+@pytest.mark.asyncio
+async def test_toggle_bad_values(config):
+    config.register_global(foo=5)
+    with pytest.raises(TypeError):
+        await config.foo.toggle()
+
+
+@pytest.mark.asyncio
+async def test_inc_bad_values(config):
+    defaults = {"foo": "bar", "baz": True}
+    config.register_global(foo=defaults)
+    with pytest.raises(TypeError):
+        await config.foo.inc(10)
+        await config.baz.inc(10)
```

Comment:
These two operations should be in separate `with` statements; `pytest.raises` will catch the first exception then exit the context manager

Reply:
Done.